editoast/src/views/infra/mod.rs

```diff
@@ -725,4 +765,26 @@ pub mod tests {
         let response = call_service(&app, delete_infra_request(infra.id.unwrap())).await;
         assert_eq!(response.status(), StatusCode::NO_CONTENT);
     }
+
+    #[actix_test]
+    async fn infra_load_core() {
+        let app = create_test_service().await;
+        let infra: Infra = call_and_read_body_json(&app, create_infra_request("lock_test")).await;
+        let mut core = MockingClient::new();
+        core.stub("/infra_load")
+            .method(reqwest::Method::POST)
+            .response(StatusCode::NO_CONTENT)
+            .body("")
+            .finish();
+        let app = create_test_service_with_core_client(core).await;
+        let req = TestRequest::post()
+            .uri(format!("/infra/{}/load", infra.id.unwrap()).as_str())
+            .to_request();
+        let response = call_service(&app, req).await;
+        println!("{:#?}", response.response());
```

Comment:
That might be a debugging leftover ;) If you want, here are a few macros that print the response content in case an assert fails here: https://github.com/DGEXSolutions/osrd/blob/c8a1770ca54769c13e51c60931707d364e1ec781/editoast/src/views/mod.rs#L93-L126

```suggestion
```